src/components/search-bar.jsx

```diff
@@ -67,13 +69,98 @@ const SearchBar = ({ mentors, setHasSearch, setSearchResults }) => {
     return () => clearTimeout(delayDebounceFn);
   }, [field, searchIndex, searchValue, setHasSearch, setSearchResults]);
 
+  const suggestions = useMemo(
+    () =>
+      [
+        ...new Set(
+          Object.values(mentors).flatMap(({ industries }) => industries)
+        ),
+      ].sort(),
+    [mentors]
+  );
+
+  useEffect(() => {
+    console.log(suggestions);
+  }, [suggestions]);
+
+  const [selectedItem, setSelectedItem] = useState([]);
+
   return (
     <form noValidate autoComplete="off" className="search-bar">
-      <TextField
-        id="standard-search"
-        label="Search mentors by..."
-        onChange={(newValue) => setSearchValue(newValue.target.value)}
-      />
+      {field === "industry" ? (
+        <Downshift
+          id="downshift-multiple"
+          onChange={(item) => {
+            setSearchValue(item);
+          }}
+          selectedItem={selectedItem}
+        >
+          {({
+            getInputProps,
+            getItemProps,
+            isOpen,
+            inputValue,
+            selectedItem,
+            highlightedIndex,
+          }) => (
+            <div className={classes.container}>
+              <TextField
+                id="standard-search"
+                label="Search mentors by..."
+                InputProps={{
+                  ...getInputProps(),
+                }}
+                onChange={(newValue) => setSearchValue(newValue.target.value)}
+                onKeyDown={(event) => {
+                  if (
+                    selectedItem.length &&
+                    !inputValue.length &&
+                    event.key === "Backspace"
+                  ) {
+                    setSelectedItem(
+                      selectedItem.slice(0, selectedItem.length - 1)
+                    );
+                  }
+                }}
+                value={searchValue}
```

Comment:
I believe that removing the `onChange` and `onKeydown` should work fine if we replace `searchValue` here with `inputValue`.